quickwit/quickwit-doc-mapper/src/default_doc_mapper/field_mapping_entry.rs

```diff
@@ -147,6 +149,7 @@ impl QuickwitTextTokenizer {
             QuickwitTextTokenizer::Default => "default",
             QuickwitTextTokenizer::StemEn => "en_stem",
             QuickwitTextTokenizer::Chinese => "chinese_compatible",
+            QuickwitTextTokenizer::Lowercase => "lowercase",
```

Comment:
I'm really afraid noone will understand that it does not tokenize.

Reply:
It's documented here: https://quickwit.io/docs/main-branch/configuration/index-config#description-of-available-tokenizers